src/fiskaltrust.Launcher/Commands/Common.cs

```diff
@@ -228,7 +230,25 @@ public static class CommonHandler
 
             return await handler(options, new CommonProperties(launcherConfiguration, cashboxConfiguration, clientEcdh, dataProtectionProvider), specificOptions, host.Services.GetRequiredService<S>());
         }
+        private static async Task EnsureServiceDirectoryExists(LauncherConfiguration config)
+        {
+            var serviceDirectory = config.ServiceFolder;
+            if (!Directory.Exists(serviceDirectory))
+            {
+                Directory.CreateDirectory(serviceDirectory);
 
+                var user = Environment.GetEnvironmentVariable("USER");
+                if (!string.IsNullOrEmpty(user))
+                {
+                    await ServiceInstaller.RunProcess("chown", new[] { user, serviceDirectory });
+                    await ServiceInstaller.RunProcess("chmod", new[] { "774", serviceDirectory });
```

Comment:
I'd move the `ServiceInstaller.RunProcess` method in its own Helper when we start using it outside of the ServiceInstaller 🙏

Also it logs the output as Information maybe debug would be nicer in this case?

We could add an optional parameter `LogEventLevel logLevel = LogEventLevel.Information` to `RunProcess` and then use `Log.Write(logLevel, stdOut)` instead of `Log.Information`
maybe same for `Log.Error` and only use warning level here 🤔

Also the method returns the exit code of the process. We should probably check it and print a warning if it fails..

Reply:
I should stop commenting solutions 🥲 you can ofc do the logging stuff however you like 😂

Reply:
I'd really move the `RunProcess` (and also use the `ProcessHelper.RunProcess` in the 
Service Installers) method instead of copying it so we don't have code duplication.

Reply:
Done 👍 I've replaced it across the whole project